internal/pkg/trustprovider/trustprovider.go

```diff
@@ -36,27 +36,18 @@ func (tp *TrustProvider) GetValues() error {
 	switch tp.Kind {
 	case "kubernetes":
 		tp.AgentConfig = TrustProviderAgentConfig{
-			WorkloadAttestor:        KubernetesTrustProvider,
-			WorkloadAttestorEnabled: true,
+			WorkloadAttestor: KubernetesTrustProvider,
 			WorkloadAttestorConfig: map[string]any{
-				"enabled":                     true,
-				"skipKubeletVerification":     true,
-				"disableContainerSelectors":   true,
-				"useNewContainerLocator":      false,
```

Comment:
👍

Reply:
I would suggest that when we upgrade we should evaluate whether we really want to diverge from the chart's defaults, or trust that they are changing the default for a good reason. Who knows - maybe the new container locator is better? :)

Reply:
That's fair enough!